### ISSUE TITLE
Command shortcut for sources list doesn't work

### ISSUE THREAD

**Issue by odisseus** [propensive/fury-old#997]
```
% fury source list
  0.006 ba6740cc//#unicorn/core
┌─────────┬─────────────────────────────┬─────────┐
│ Repo    │ Path                        │ Sources │
├─────────┼─────────────────────────────┼─────────┤
│ local   │ src                         │ **/*    │
│ unicorn │ unicorn-core/src/main/scala │ **/*    │
└─────────┴─────────────────────────────┴─────────┘
% fury sl
  0.009 ba6740cc//#unicorn/core
┌──────┬──────┬───────────┐
│ Repo │ Base │ Resources │
├──────┼──────┼───────────┤
└──────┴──────┴───────────┘
```

**Comment by propensive** [comment]
There's a conflict with `resource`. I'll fix it.

**Comment by propensive** [comment]
Should be fixed in #1001, but it's untested.
